### ISSUE TITLE
Invalid trigger data when converting EVIO data

### ISSUE THREAD

**Issue by andrea-celentano** [JeffersonLab/hps-java#465]
When trying to reconstruct data from a ECAL-only run taken with the 2019 DAQ, the following error is reported at every event.

```
019-05-07 09:39:41 [SEVERE] org.hps.evio.LCSimTestRunEventBuilder getTriggerData :: null
java.lang.reflect.InvocationTargetException
    at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
    at org.hps.evio.LCSimTestRunEventBuilder.getTriggerData(LCSimTestRunEventBuilder.java:173)
    at org.hps.evio.LCSimTestRunEventBuilder.getEventData(LCSimTestRunEventBuilder.java:115)
    at org.hps.evio.LCSimEngRunEventBuilder.makeLCSimEvent(LCSimEngRunEventBuilder.java:143)
    at org.hps.evio.EvioToLcio.run(EvioToLcio.java:608)
    at org.hps.evio.EvioToLcio.main(EvioToLcio.java:92)
Caused by: java.lang.RuntimeException: Invalid Data Length:  6
    at org.hps.record.triggerbank.HeadBankData.decodeData(HeadBankData.java:76)
    at org.hps.record.triggerbank.HeadBankData.<init>(HeadBankData.java:23)
    ... 8 more
```

The run number is "81", this causes hps-java to use the eng. run event builder. It looks the "-R" option passed on the command line is not changing this.

**Comment by JeremyMcCormick** [comment]
> The run number is "81", this causes hps-java to use the eng. run event builder. It looks the "-R" option passed on the command line is not changing this.

@andrea-celentano You need to include the exact command line you are running in all your bug reports, so that the problem can be reproduced easily.

I'll look more closely at this error.  It seems like a data format change to the head bank in EVIO but I need to confirm.

**Comment by JeremyMcCormick** [comment]
This should be fixed by PR #469.
